endpoint_info.go

```diff
@@ -227,7 +232,17 @@ func (ep *endpoint) DriverInfo() (map[string]interface{}, error) {
 		return nil, fmt.Errorf("failed to get driver info: %v", err)
 	}
 
-	return driver.EndpointOperInfo(n.ID(), ep.ID())
+	epInfo, err := driver.EndpointOperInfo(n.ID(), ep.ID())
+	if err != nil {
+		return nil, err
+	}
+
+	if epInfo != nil {
+		epInfo["GW_INFO"] = gwDriverInfo
+		return epInfo, nil
+	}
+
+	return gwDriverInfo, nil
```

Comment:
In linux as well, daemon is not aware of the second interface in the sandbox which provides the external connectivity.
Oh, you are saying the windows overlay driver will query the daemon to get the information about the default gw endpoint ?